flow/core/kernel/vehicle/traci.py

```diff
@@ -1195,3 +1217,17 @@ def get_road_grade(self, veh_id):
         """See parent class."""
         # TODO : Brent
         return 0
+
+    def get_soc(self, veh_id):
+        return self.__vehicles[veh_id]['SOC']
+
+    def set_soc(self, veh_id, soc):
+        """Set the lane headways of the specified vehicle."""
+        self.__vehicles[veh_id]['SOC'] = soc
+
+    def get_fuel(self, veh_id):
+        return self.__vehicles[veh_id]['fuel']
+
+    def set_fuel(self, veh_id, fuel):
+        """Set the lane headways of the specified vehicle."""
+        self.__vehicles[veh_id]['fuel'] = fuel
```

Comment:
see above comment, i think we can delete these.

Reply:
How to add an electric vehicle and get ElectricityConsumption